netket/hilbert/random/fermions2nd.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 The NetKet Authors - All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import jax
+from jax import numpy as jnp
+
+from netket.hilbert import SpinOrbitalFermions, OrbitalFermions
+from netket.utils.dispatch import dispatch
+
+
+@dispatch
+def random_state(hilb: OrbitalFermions, key, batches: int, *, dtype):
+    return random_state(hilb._fock, key, batches, dtype)
+
+
+@dispatch
+def flip_state_scalar(hilb: OrbitalFermions, key, state, index):
+    return flip_state_scalar(hilb._fock, key, state, index)
+
```

Comment:
it might be worth it to also forward `flip_state_batch`. might give you a negligible speedup

Reply:
TensorHilbert and Fock don't seem to have this?